website/index.html

```diff
@@ -21,87 +21,94 @@ <h2>
             </a>
         </header>
         <div class="flex-all-content">
```

Comment:
We usually do not name a class with what it does, rather we want to describe it by what it is.
```suggestion
        <div class="content"> or "main-content"
```

Reply:
2898f50

Reply:
U missed `.text-image-flex`

Reply:
bb83bf3

Reply:
6f21875